src/openrct2/ride/TrackData.cpp

```diff
@@ -5658,7 +5788,7 @@ static constexpr rct_trackdefinition TrackDefinitions[TrackElemType::Count] =
     {TRACK_MINI_GOLF_HOLE,          TRACK_SLOPE_NONE,    TRACK_SLOPE_NONE,  TRACK_BANK_NONE,        TRACK_BANK_NONE,        0                  },  // ELEM_MINI_GOLF_HOLE_C
     {TRACK_MINI_GOLF_HOLE,          TRACK_SLOPE_NONE,    TRACK_SLOPE_NONE,  TRACK_BANK_NONE,        TRACK_BANK_NONE,        0                  },  // ELEM_MINI_GOLF_HOLE_D
     {TRACK_MINI_GOLF_HOLE,          TRACK_SLOPE_NONE,    TRACK_SLOPE_NONE,  TRACK_BANK_NONE,        TRACK_BANK_NONE,        0                  },  // ELEM_MINI_GOLF_HOLE_E
-    {TRACK_QUARTER_LOOP_INVERTED,   TRACK_SLOPE_DOWN_90, TRACK_SLOPE_NONE,  TRACK_BANK_UPSIDE_DOWN, TRACK_BANK_NONE,        0                  },  // ELEM_MULTIDIM_INVERTED_FLAT_TO_90_DEG_QUARTER_LOOP_DOWN
+    {TRACK_QUARTER_LOOP_INVERTED_DOWN,   TRACK_SLOPE_DOWN_90, TRACK_SLOPE_NONE,  TRACK_BANK_UPSIDE_DOWN, TRACK_BANK_NONE,        0                  },  // ELEM_MULTIDIM_INVERTED_FLAT_TO_90_DEG_QUARTER_LOOP_DOWN
```

Comment:
Could you explain why this occurrence had to be replaced with `TRACK_QUARTER_LOOP_INVERTED_DOWN`, while the other location (on line 5840) was replaced with `TRACK_QUARTER_LOOP_INVERTED_UP`. Is this to support both types within one roller coaster, or?

Reply:
It was so that flying coaster inverted can build FlatToDown90Inverted and not Up90ToFlatInverted and flying coaster upright can build Up90ToFlatInverted and not FlatToDown90Inverted.